### DISCUSSION TITLE
Where is srt?

### DISCUSSION THREAD

**Discussion by FibaGold** [EtienneAb3d/WhisperHallu#25] [Q&A]
I'm new to all of this I try this one on Google Colab "Standard Whisper" 
after all install I just run the script:

```
from transcribeHallu import loadModel
from transcribeHallu import transcribePrompt

##### The audio language may be different from the one for the output transcription.
path="/path/to/your/en/sound/file"
lngInput="ja"


##### Activate this for music file to get a minimal processing
isMusic=False

##### Need to be adapted for each language.
##### For prompt examples, see transcribeHallu.py getPrompt(lng:str)
lng="ja"
prompt= "Whisper, Ok. "\
	+"A pertinent sentence for your purpose in your language. "\
	+"Ok, Whisper. Whisper, Ok. "\
	+"Ok, Whisper. Whisper, Ok. "\
	+"Please find here, an unlikely ordinary sentence. "\
	+"This is to avoid a repetition to be deleted. "\
	+"Ok, Whisper. "

##### Model size to use
modelSize="medium"
loadModel("0",modelSize=modelSize)

result = transcribePrompt(path=path, lng=lng, prompt=prompt, lngInput=lngInput,isMusic=isMusic)
```
![Screenshot_1](https://github.com/EtienneAb3d/WhisperHallu/assets/156567552/a06cb27f-b5a9-4909-b849-5fd354f36f82)

but it didn't give me any srt or txt format...it just give me audio that was cut off the selience from original source...What I need to add to give me srt or txt of that audio?

**Comment by EtienneAb3d** [comment]
Print the "result" value.
;-)

**Reply by FibaGold** [reply]
![Screenshot_2](https://github.com/EtienneAb3d/WhisperHallu/assets/156567552/4c7a8031-b94a-4b92-9101-f440b49a33d3)
![Screenshot_3](https://github.com/EtienneAb3d/WhisperHallu/assets/156567552/aff505a4-9c88-4f53-a1a3-fe7012a9eda2)

**Comment by EtienneAb3d** [comment]
In Your code, you have this:

`result = transcribePrompt(path=path, lng=lng, prompt=prompt, lngInput=lngInput,isMusic=isMusic)
`

The **`result`** variable contains the transcription.

**Comment by FibaGold** [comment]
> In Your code, you have this:
> 
> `result = transcribePrompt(path=path, lng=lng, prompt=prompt, lngInput=lngInput,isMusic=isMusic) `
> 
> The **`result`** variable contains the transcription.

so what I need to change or to add to give me the srt or txt file?

**Reply by EtienneAb3d** [reply]
You need to write it to a file.
https://www.w3schools.com/python/python_file_write.asp

WhisperHallu is an experimental code. You need minimal developer skills to understand it and use it.
